vacation/src/main/java/cz/fi/muni/pv217/employee/management/service/VacationService.java

```diff
@@ -0,0 +1,44 @@
+package cz.fi.muni.pv217.employee.management.service;
+
+import cz.fi.muni.pv217.employee.management.entity.Vacation;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.transaction.Transactional;
+import javax.ws.rs.NotFoundException;
+
+@ApplicationScoped
+public class VacationService {
+
+    @Transactional
+    public Vacation createVacation(Vacation vacation) {
+        vacation.persist();
+        return vacation;
+    }
+
+    @Transactional
+    public Vacation updateVacation(long id, Vacation changedVacation) {
+        Vacation vacation = Vacation.findById(id);
+
+        if (vacation == null) {
+            return null;
```

Comment:
Seems a little inconsistent in delete method you throw NotFoundException but here you throw null.